database/reindex_elastic.js

```diff
@@ -91,12 +70,10 @@ const indexEntities = async () => {
 const prepareIndex = async () => {
   process.stdout.write(`Deleting index ${tenants.current().indexName}...`);
   try {
-    await fetch(getIndexUrl(), { method: 'delete' });
+    await elastic.indices.delete({});
   } catch (err) {
-    // Should not stop on index_not_found_exception
-    if (err.json.error.type === 'index_not_found_exception') {
-      process.stdout.write('\r\nThe index was not found:\r\n');
-      process.stdout.write(`${JSON.stringify(err, null, ' ')}\r\nMoving on.\r\n`);
+    if (err?.meta?.body?.error?.type === 'index_not_found_exception') {
```

Comment:
How do you get to use TS '?' operators in a JS file?

Reply:
Optional chaining operator is a JS operator introduced in ecma2020 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining